leader-election-impl/src/main/java/com/palantir/paxos/SingleLeaderPinger.java

```diff
@@ -287,21 +286,12 @@ private void throwIfInvalidSetup(
             return;
         }
 
-        IllegalStateException exception =
```

Comment:
I am not familiar with the *.pref things referenced here. I am guessing these are outdated and relate to previous deployment models?

Reply:
Yeah I'm also unfamiliar with this and am fairly certain this message is no longer relevant with our current deployment models.